plugins/amazonq/mynah-ui/src/mynah-ui/ui/tabs/generator.ts

```diff
@@ -49,6 +50,13 @@ export class TabDataGenerator {
 Here I can provide code suggestions across files in your current project by looking at /src, if it exists.
 
 Before I begin generating code, let's agree on an implementation plan. What change would you like to make?
+`,
+        ],
+        [
+            'codetransform',
+            `Welcome to Code Transformation!
+
+I can help you upgrade your Java 8 and 11 codebases to Java 17.
```

Comment:
@rli 
Where do you want UI Text for typescript?

Reply:
we can leave these in TS

Reply:
I don't think we currently translate strings in ts files?